### DISCUSSION TITLE
Auto-alphebetize home screen setting

### DISCUSSION THREAD

**Discussion by paerbac** [tanujnotes/Olauncher#229] [Ideas]
I would think it would be really cool to have this feature so I don't have to resort all my apps every time i decide to change up my home screen

**Comment by Heike-HS** [comment]
Whereas I do not want apps on Homescreen in alphabetic order. I sort then rather by functionality.

**Reply by paerbac** [reply]
You can already do that, I was suggesting a switch or toggle that determines whether it's alphabetical or not, I wasn't suggesting it to be the default
